src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
```

Comment:
@davidsh, I'm unclear as to what the right default should be for HttpClientHandler.UseCookies.  It looks like WinHttpHandler maps UseCookies to _cookieUsePolicy, which WinHttpClientHandler defaults to UseInternalCookieStoreOnly, but then the wrapping HttpClientHandler's UseCookies will by default return false, because its getter compares it to UseSpecifiedCookieContainer (https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs#L35).  So should CurlHandler.UseCookies default to false instead of defaulting to true as it currently is?